.circleci/config.yml

```diff
@@ -0,0 +1,87 @@
+# These environment variables must be set in CircleCI UI
+#
+# DOCKERHUB_REPO - docker hub repo, format: <username>/<repo>
+# DOCKER_EMAIL   - login info for docker hub
+# DOCKER_USER
+# DOCKER_PASS
+#
+version: 2
+jobs:
+  build:
+    docker:
+      # base image
+      - image: ubuntu:16.04
+    steps:
+      - checkout
+      - setup_remote_docker
+      - run:
+          name: Install essential packages
+          command: |
+            apt-get update
+            apt-get install -y ca-certificates curl build-essential make git
+
+      - run:
+          name: Create version.json
+          command: |
+            printf '{"commit":"%s","version":"%s","source":"https://github.com/%s/%s","build":"%s"}\n' \
+            "$CIRCLE_SHA1" \
+            "$CIRCLE_TAG" \
+            "$CIRCLE_PROJECT_USERNAME" \
+            "$CIRCLE_PROJECT_REPONAME" \
+            "$CIRCLE_BUILD_URL" > version.json
+      - store_artifacts:
+          path: version.json
+      # FIXME: should use an image w/ docker installed by default
+      - run:
+          name: Install Docker
+          command: |
+            set -x
+            curl -L -o /tmp/docker-$DOCKER_VERSION.tgz https://download.docker.com/linux/static/stable/x86_64/docker-$DOCKER_VERSION.tgz
+            tar -xz -C /tmp -f /tmp/docker-$DOCKER_VERSION.tgz
+            mv /tmp/docker/* /usr/bin
+      - run:
+          name: Build deployment container image
+          command: docker build -t app:build .
+      - run:
+          name: Create Docker image shasum256
+          command: |
+            docker images --no-trunc |
```

Comment:
@autrilla this looks wrong, this should instead get the digest via something like `docker images --digests`, the IMAGE ID is not the number we want

Reply:
hm for me it shows `IMAGE ID`s, those are not the same as digest SHAs, do you get digest numbers in your case?

Reply:
Example

```
 docker images --no-trunc --digests
REPOSITORY           TAG                 DIGEST                                                                    IMAGE ID                                                                  CREATED             SIZE
redis                <none>              sha256:2a10443dced488731f0a3ed12e442c230d39b526f95ecb8e280cd1596705e822   sha256:c5355f8853e4174a55144edfec23ac37f5bb2200ed838dab53c13c7cc835ab15   6 weeks ago         107 MB
```

Reply:
@jbuck ideas?

Reply:
You're right, I get a different digest from the image id too. Not sure why we would use one over the other, though? Is there a reason we should keep using the image ID? It's what it was using in the past so I have no idea if there's a reason for it.

Reply:
You're right, I get a different digest from the image id too. Not sure why we would use one over the other, though? Is there a reason we should keep using the image ID? It's what it was using in the past so I have no idea if there's a reason for it.

Reply:
I think the digest is calculated based on some metadata when you actually push the image to a repo